packages/bigquery/src/table.js

```diff
@@ -473,7 +459,7 @@ Table.prototype.createWriteStream = function(metadata) {
  * @throws {Error} If destination format isn't recongized.
  *
  * @example
- * var gcs = gcloud.storage({
+ * var gcs = require('@google-cloud/storage')({
```

Comment:
This is a BigQuery API that is using storage. Let me know if I misunderstood your question.